studentInfo.py

```diff
@@ -0,0 +1,72 @@
+  fullItems = ['Abyssal Mask','Adaptive Helm','Archangels Staff','Ardent Censer','Unholy Grail','Banner of Command','Banshees Veil','Blade of the Ruined King','Dead Mans Plate','Deaths Dance','Duskblade of Drakathaar','Edge of Night','Essence Reaver','Eye of Equinox','Eye of the Oasis','Eye of the Watcher','Face of the Mountain','Frost Queens Claim','Frozen Heart','Frozen Mallet','Gargoyle Stoneplate','Guardian Angel','Guinsoos Rageblade','Hextech GLP-800','Hextech Gunblade','Hextech Protobelt-01','Iceborn Gauntlet','Infinity Edge','Knights Vow','Liandrys Torment','Lich Bane','Locket of the Iron Solari','Lord Dominiks Regards','Ludens Echo','Manamune','Maw of Malmortius','Mejais Soulstealer','Mercurial Scimitar','Mikaels Crucible','Morellonomicon','Mortal Reminder','Nashors Tooth','Ohmwrecker','Phantom Dancer','Rabadons Deathcap','Randuins Omen','Rapid Firecannon','Ravenous Hydra','Redemption','Righteous Glory','Rod Of Ages','Ruby Sightstone','Runaans Hurricane','Rylais Crystal Scepter','Spirit Visage','Statikk Shiv','Steraks Gage','Sunfire Cape','Talisman of Ascension','The Black Cleaver','The Bloodthirstier','Thornmail','Titanic Hydra','Trinity Force','Void Staff','Warmogs Armor','Wits End','Yomuus Ghostblade','Zekes Convergence','Zhonyas Hourglass','ZZ-Rot Portal','','','','','','','','']
+  def __init__(self):
+    self.assignRandomK()
+    self.assignRandomD()
+    self.assignRandomA()
+    self.randomChampion()
+    self.randomLane()
+    self.randomItem1()
+    self.randomItem2()
+    self.randomItem3()
+    self.randomItem4()
+    self.randomItem5()
+    self.randomItem6()
+
+  def randomLane(self):
+     self.lane = random.choice(self.lanes)
+
+  def randomChampion(self):
+     self.mainChamp = random.choice(self.mainChamps)
+  
+  def assignRandomK(self):
+     self.K = random.randint(0,30)
+  def assignRandomD(self):
+     self.D = random.randint(0,30)
+  def assignRandomA(self):
+     self.A = random.randint(0,30)  
+  def randomItem1(self):
+    self.item1 = random.choice(self.fullItems)
+  def randomItem2(self):
+    self.item2 = random.choice(self.fullItems)  
+  def randomItem3(self):
+    self.item3 = random.choice(self.fullItems)
+  def randomItem4(self):
+    self.item4 = random.choice(self.fullItems)
+  def randomItem5(self):
+    self.item5 = random.choice(self.fullItems)
+  def randomItem6(self):
+    self.item6 = random.choice(self.fullItems)
+
+inputQuestions = [ 
+  "For CHAMPIONS, type 0",
+]
+
+def getUserSelection():
+  print (inputQuestions[0])
+  return int(input("Type selection and press enter: "))
+
+def RandomizedChampions():
+  count = int( input("How Many Randomized Champions? "))
+  for i in range(0, count):
+    champ = Champion()
+    print (str(champ.mainChamp) + " " + str(champ.lane) + ", " + str(champ.item1) + ", " + str(champ.item2) + ", " + str(champ.item3) + ", " + str(champ.item4) +", " + str(champ.item5) + ", " + str(champ.item6))
```

Comment:
In regards to 

```
print (str(champ.mainChamp) + " " + str(champ.lane) + ", " + str(champ.item1) + ", " + str(champ.item2) + ", " + str(champ.item3) + ", " + str(champ.item4) +", " + str(champ.item5) + ", " + str(champ.item6))
```

You don't need to wrap champ.mainChamp or any of the other values in `str()` as they're already strings.

```
print (champ.mainChamp + " " + champ.lane + ", " + champ.item1 + ", " + champ.item2 + ", " + champ.item3 + ", " + champ.item4 +", " + champ.item5 + ", " + champ.item6)
````

It's also more legible :+1: 

"Proof":

```
jcharante@Thinky:~$ python2.7
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> class A:
...     def __init__(self):
...             self.name = "bob"
... 
>>> a = A()
>>> print(a.name + " is the value of name for an instance of class A")
bob is the value of name for an instance of class A
```


You could also try other forms of string interpolation which could be more legible.

```
# Without str() wrappers and addition
print (champ.mainChamp + " " + champ.lane + ", " + champ.item1 + ", " + champ.item2 + ", " + champ.item3 + ", " + champ.item4 +", " + champ.item5 + ", " + champ.item6)

# With the modulo operator (the most recognizable one since it's implemented in many languages)
print("%s %s, %s, %s, %s, %s, %s, %s" % (champ.mainChamp, champ.lane, champ.item1, champ.item2, champ.item3, champ.item4, champ.item5, champ.item6))

# With str's .format()
print("{champ} {lane}, {item1}, {item2}, {item3}, {item4}, {item5}, {item6}"\
    .format(
        champ=champ.mainChamp,
        lane=champ.lane,
        item1=champ.item1,
        item2=champ.item2,
        item3=champ.item3,
        item4=champ.item4,
        item5=champ.item5,
        item6=champ.item6
    )
)

# Python 3.6's f strings
print(f"{champ.mainChamp} {champ.lane}, {champ.item1}, {champ.item2}, {champ.item3}, {champ.item4}, {champ.item5}, {champ.item6}")
```